funflow/src/Control/FunFlow/ContentHashable.hs

```diff
@@ -439,3 +445,61 @@ instance ContentHashable IO DirectoryContent where
         flip contentHashUpdate (Path.dirname dir)
         >=> flip contentHashUpdate (DirectoryContent dir)
         $ ctx
+
+instance Monad m => ContentHashable m UTCTime where
+  contentHashUpdate ctx utcTime = let
+      secondsSinceEpoch = fromEnum . utcTimeToPOSIXSeconds $ utcTime
+    in contentHashUpdate ctx secondsSinceEpoch
```

Comment:
Should also use `contentHashUpdate_fingerprint` to distinguish this from an `Int` `ContentHash`.